src/routes/Router/Router.tsx

```diff
@@ -19,36 +22,52 @@ export const Router = createBrowserRouter(
       errorElement={<NotFoundSite />}
     >
       <Route
-        path={RouterPath.OfferedApiDelegations + '/' + RouterPath.Overview}
+        path={ApiDelegationPath.OfferedApiDelegations}
         element={<OfferedOverviewPage />}
         errorElement={<NotFoundSite />}
-      />
+      >
+        <Route
+          path={ApiDelegationPath.Overview}
```

Comment:
Shouldn't they follow this pattern?

```suggestion
          path={ApiDelegationPath.OfferedApiDelegations + '/' + ApiDelegationPath.Overview}
```